pkg/controller/operators/olm/operator.go

```diff
@@ -855,15 +849,18 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 		}
 
 		// Check for CRD ownership conflicts
-		csvSet := a.csvSet(out.GetNamespace(), v1alpha1.CSVPhaseAny)
-		if syncError = a.crdOwnerConflicts(out, csvSet); syncError != nil {
-			out.SetPhaseWithEvent(v1alpha1.CSVPhaseFailed, v1alpha1.CSVReasonOwnerConflict, fmt.Sprintf("crd owner conflict: %s", syncError), now, a.recorder)
+		if syncError = a.crdOwnerConflicts(out, a.csvSet(out.GetNamespace(), v1alpha1.CSVPhaseAny)); syncError != nil {
```

Comment:
nit: even nicer would be to just generate the csv set in the method.

Reply:
Doing that makes the function less _pure_ , which makes it slightly more difficult to unit test. We would need to populate the client/informer/listers with test data  instead of just passing CSVs in.

Disclaimer: not sure if I have a unit test for this one.

Reply:
Guess I'm already querying for replacements in the method anyway huh.